independent-publisher-connectors/Cisco Webex List Meeting/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
```

Comment:
@last72 in the swagger file you are mentioning OAuth type. can you please fix the apiProperties.json file as well?